scripts/dog-info.js

```diff
@@ -0,0 +1,28 @@
+let dogName;
+let fees = [];
+let fee = 0;
+let total = 0;
+
+function viewDogInfo(dogName, breed, fee) {
+  let dogInfo = 'Name: ' + dogName  + '\n' + 'Breed: ' + breed + '\n' + 'Fee: $' + fee;
+  alert(dogInfo);
+}
+
+function addFee(dogName, fee, id, checkoutTotal) {
+  fee = Number(fee);
+  fees.push(fee);
+  let total = fees.reduce(function(a, b) {
```

Comment:
Great work tying in some JS functions! You'll do some more work with these in HTML 300!